### PR TITLE
DTGB-433: Fixed translations.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: translations/general.pot [hunk 1]
```diff
@@ -29,7 +30,11 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
-"POT-Creation-Date: 2018-06-19 08:58+0000\n"
+<<<<<<< Updated upstream
```
Comment (delrueba) [inline translations/general.pot]: Merge conflict

File: translations/nl.po [hunk 2]
```diff
@@ -24,8 +24,13 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
-"POT-Creation-Date: 2018-06-19 08:58+0000\n"
-"PO-Revision-Date: 2018-06-19 11:01+0200\n"
+<<<<<<< Updated upstream
```
Comment (delrueba) [inline translations/nl.po]: Merge conflict

**Review by delrueba — CHANGES_REQUESTED** [review]

File: translations/general.pot (hunk 1 above)
Comment (meirege) [reply translations/general.pot]: Fixed.

File: translations/nl.po (hunk 2 above)
Comment (meirege) [reply translations/nl.po]: Fixed.

**Review by delrueba — APPROVED** [review]

**Pull request merged**